plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -3484,6 +3483,31 @@ eosio::chain::backing_store_type read_only::get_backing_store() const {
 }
 
 } // namespace chain_apis
+
+fc::variant chain_plugin::get_entire_trx_trace(const std::variant<fc::exception_ptr, transaction_trace_ptr, std::shared_ptr<transaction>>& obj)const {
```

Comment:
This method is misnamed now that it does 3 diff things.

Reply:
Fixed.